components/Layout/index.tsx

```diff
@@ -11,31 +12,27 @@ export default ({ children }: Props) => {
     <>
       <Head>
         <title>Golden Rune Calculator</title>
-        {/*  Global site tag (gtag.js) - Google Analytics */}
-        {/* eslint-disable-next-line @next/next/next-script-for-ga */}
-        <script
-          async
-          src="https://www.googletagmanager.com/gtag/js?id=G-KCTJK4JDN7"
-        />
-        <script
-          dangerouslySetInnerHTML={{
-            __html: `
-                <script>
-                  window.dataLayer = window.dataLayer || [];
-                  function gtag(){dataLayer.push(arguments);}
-                  gtag('js', new Date());
-                
-                  gtag('config', 'G-KCTJK4JDN7');
-                </script>`,
-          }}
-        />
-        {/* End global site tag (gtag.js) - Google Analytics */}
         <meta
           name="description"
           content="Golden Rune Calculator for Elden Ring"
         />
         <link rel="icon" href="/favicon.png" />
       </Head>
+      {/*  Global site tag (gtag.js) - Google Analytics */}
+      <Script
```

Comment:
@sergnio I was seeing some warnings from Next saying to use their Script tag. I found [an example on their site](https://nextjs.org/docs/messages/next-script-for-ga) for Google Analytics.